dspace/config/clarin-dspace.cfg

```diff
@@ -282,4 +282,7 @@ download.all.limit.max.file.size = 1073741824
 # minimum total size of files for enabling download alert:
 download.all.alert.min.file.size = 10485760
 # used in elg crosswalk exposing download locations
-elg.download-location.exposed = 0
\ No newline at end of file
+elg.download-location.exposed = 0
+
+#download.email.cc = ${info.recipient}
```

Comment:
?

Reply:
info.recipient was used in ufal, but no longer exists. Left this here as reminder, if someone knew and was searching for it.
What is better approach?

Reply:
improve docs with info from your reply